central/networkgraph/flow/datastore/internal/store/postgres/tests/store_test.go

```diff
@@ -76,33 +76,37 @@ func (s *NetworkflowStoreSuite) TestStore() {
 	secondCluster := fixtureconsts.Cluster2
 	store2 := postgresFlowStore.New(s.pgDB.DB, secondCluster)
 
+	now := time.Now().UTC()
```

Comment:
Was failing locally without `UTC()`, even though it was passing in CI. It seems to be some kind of problem with conversion and time zone.

Reply:
That doesn't make much sense.  Now we have seen where the time will get rounded in one place and not the other presumably due to architecture running the test.  You may want to verify that that may be your case vs UTC.  Stripping nanos would make more sense if required.

Reply:
I have changed it to `now := time.Now().UTC().Truncate(time.Second)` so there should not be a problem with rounding nanoseconds. The test still failed for me when I changed it to `now := time.Now().Truncate(time.Second)`.